src/dotnet-interactive/Connection/StdIoKernelConnector.cs

```diff
@@ -60,17 +60,6 @@ public class StdIoKernelConnector : IKernelConnector
         _remoteKernelInfoCache = new ConcurrentDictionary<string, KernelInfo>();
     }
 
-    /// <remarks>
-    /// TODO: Does it even make sense to implement <see cref="IKernelConnector"/> here considering that we have
-    /// concepts (such as a single root <see cref="ProxyKernel"/>, and zero or more (child) <see cref="ProxyKernel"/>s)
-    /// that the <see cref="IKernelConnector"/> abstraction does not understand / support? Should the '#!connect stdio'
```

Comment:
Just curious - any changes in our thoughts around the `#!connect stdio`? Can we remove that too?

Reply:
I think we might still want this, especially as project integration comes along and we think about .NET Interactive instances for other TFMs, different dependency graphs, etc.